src/inject/style.ts

```diff
@@ -9,6 +10,15 @@ export function createOrUpdateStyle(css: string) {
             style.type = 'text/css';
             style.textContent = css;
             target.appendChild(style);
+            const wait = (animationFrame: number) => requestAnimationFrame(() => {
```

Comment:
What's the purpose of this? Wait for 60 frames (about 1 second)? I think just removing the fallback should work.

Reply:
AFAIK laptops with integrated GPU onto the CPU takes more frames to get the filter applied over it and rendered. Also not sure how much it, it improves it due to the html still having a white background.